File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/frontend/src/app/models/sto.ts
export class Sto
{
    public id: number;
    public kaficId: number;
    public korisnikId: number;
    public brojOsoba: number;
    public rezervisan: string;
    public username: string

    constructor(id: number, kaficId: number, brojOsoba: number, rezervisan: string)
    {
        this.id = id;
        this.kaficId = kaficId;
        this.korisnikId = 0;
        this.brojOsoba = brojOsoba;
        this.rezervisan = rezervisan;
        this.username = "";
    }
}

File: wlada-si.21.19.getout-e707bbd07835/Aplikacija/SWEproj/backend/GETOut/Models/Sto.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

#nullable disable

namespace GETOut.Models
{
    [Table("sto")]
    public partial class Sto
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("kafic_id")]
        public int KaficId { get; set; }

        [Column("username")]
        public string Username { get; set; }

        [Column("broj_osoba")]
        public int BrojOsoba { get; set; }

        [Column("rezervisan")]
        public string Rezervisan { get; set; }

    }
}
